plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/dataexport/AbstractTypeExporter.java

```diff
@@ -192,8 +193,12 @@ private void addImport(final Import imp) throws XMLStreamException {
 	 * @param varDecl the var decl
 	 * @throws XMLStreamException
 	 */
+	@SuppressWarnings("unlikely-arg-type")
 	protected void addVarDeclaration(final VarDeclaration varDecl) throws XMLStreamException {
-		final boolean hasAttributes = !varDecl.getAttributes().isEmpty();
+		final boolean hasAttributes = !varDecl.getAttributes().isEmpty()
+				|| (varDecl.isInOutVar() && !varDecl.getInOutVarOpposite().getAttributes().isEmpty());
+		final boolean hasOutAttributes = varDecl.getAttributes().stream().map(Attribute::getName).toList()
+				.contains("VisibleOutSide"); //$NON-NLS-1$
```

Comment:
Usally there should be a constant for that somewhere in 4diac.
Please try to use a workspace text search und check if there exists a constant for "VisibleOutSide"

Reply:
This is a new entry. it should be add to our constants.